tf_encrypted/keras/layers/dense.py

```diff
@@ -0,0 +1,121 @@
+# pylint: disable=arguments-differ
+"""Dense (i.e. fully connected) Layer implementation."""
+from typing import Union, Optional
+import numpy as np
+import tensorflow as tf
+from tensorflow.python.keras import initializers
+
+from tf_encrypted.keras.engine.base_layer import Layer
```

Comment:
the less specific `from tf_encrypted.keras.engine import Layer` should work as well